### DISCUSSION TITLE
Typescript intellisense

### DISCUSSION THREAD

**Discussion by larisateodor2** [AstrOOnauta/react-native-international-phone-number#5] [Q&A] [answered]
I'm usying the new version (0.4.13) and the component `PhoneInput` don't show the typescript intellisense.
Print for this case below:

![image](https://github.com/AstrOOnauta/react-native-international-phone-number/assets/141968183/1f9e8c30-d55c-4724-82ef-ddec44a2619b)

Please, can anyone help me?

**Comment by AstrOOnauta** [comment, marked as answer]
Hey @larisateodor2 

Great observation!
The version 0.4.13 breaks the lib typescript intellisense, sorry for this.
I fixed this issue in the next version (v0.4.13). So, please update the version of your lib `react-native-international-phone-number` to `v0.4.14` and enjoy.

🚀AstrOOnauta 🚀
